esp-hal/src/interrupt/mod.rs

```diff
@@ -85,13 +85,6 @@ mod xtensa;
 
 pub mod software;
 
-#[cfg(xtensa)]
-#[no_mangle]
-extern "C" fn EspDefaultHandler(_level: u32, _interrupt: crate::peripherals::Interrupt) {
```

Comment:
Nice, this asymmetry has bothered me for the longest time, and it's so easy to get the current level there was never any point in passing it around.